src/model/json-schema.json

```diff
@@ -174,6 +174,9 @@
         "webPublicationDateDisplay": {
             "type": "string"
         },
+        "webPublicationSecondaryDateDisplay": {
```

Comment:
Is this a new field we're adding to the model in frontend?

Reply:
Yes, I've already added it